miner/worker.go

```diff
@@ -1080,6 +1081,32 @@ func (w *worker) fillTransactions(interruptCh chan int32, env *environment, stop
 			localBlobTxs[account] = txs
 		}
 	}
+
+	if bidTxs != nil {
+		var wg sync.WaitGroup
+		wg.Add(4)
+
+		filterBidTxs := func(commonTxs map[common.Address][]*txpool.LazyTransaction) {
+			defer wg.Done()
+			for acc, txs := range commonTxs {
+				newTxs := make([]*txpool.LazyTransaction, 0)
+				for _, tx := range txs {
+					if !bidTxs.Contains(tx.Hash) {
+						newTxs = append(newTxs, tx)
+					}
+				}
+				commonTxs[acc] = newTxs
```

Comment:
I think in most cases the txhash will not conflict, we dont have to do `commonTxs[acc] = newTxs` for each account. Just assign only if necessary?

Reply:
fixed